lms/envs/common.py

```diff
@@ -2712,3 +2712,7 @@
 # The reason we introcuced this number is because we do not want the CCX
 # to compete with the MOOC.
 CCX_MAX_STUDENTS_ALLOWED = 200
+
+#### Registration form extension. Only used if combined login/registration is enabled ####
```

Comment:
Can you expand this slightly? Maybe say that you can override with a custom django form, see blah for example, and that this must be the python path to a valid django form (with the current implementation it can't be None I think, right), and that the default value is a form with no custom fields.